metrics_handler/requirements.txt

```diff
@@ -5,5 +5,5 @@ google-cloud-bigquery==1.23.1
 google-cloud-error-reporting==0.33.0
 google-cloud-pubsub==1.2.0
 google-cloud-storage==1.23.0
-tensorboard>=2.0.0
-tensorflow>=1.14.0
+tensorboard==2.0.0
+tensorflow==1.14.0
```

Comment:
Do you need these specific versions? If you set them like this, you should set them to corresponding releases (e.g. set both to `2.1.0`)